start

```diff
@@ -53,6 +54,10 @@ function main() {
     echo "--randomize-network-passphrase is only supported in the standalone network" >&2
     exit 1
   fi
+  if [ "$NETWORK" != "standalone" ] && [ "$ENABLE_SOROBAN_HIGH_LIMIT_OVERRIDE" = "true" ]; then
```

Comment:
might be more apt to validate usage with `$ENABLE_SOROBAN_RPC" != "true"`? regardless of network.

Reply:
I don't think so. This feature is accessible independent of whether the soroban-rpc is running.